src/main/java/qna/config/JpaAuditingConfig.java

```diff
@@ -0,0 +1,9 @@
+package qna.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
+
+@EnableJpaAuditing
+@Configuration
+public class JpaAuditingConfig {
```

Comment:
```suggestion
public class JpaConfig {
```

지금의 이름으론 설정이 추가될 때 마다 Config가 추가되겠네요!
Jpa에 대한 설정은 하나의 Config에서 해주면 어떨까요?